ipatests/test_integration/test_commands.py

```diff
@@ -133,3 +133,48 @@ def test_change_sysaccount_password_issue7561(self):
 
         tasks.ldappasswd_sysaccount_change(sysuser, original_passwd,
                                            new_passwd, master)
+
+
+class TestIPACAInstallCommand(IntegrationTest):
+    """ ipa-ca-install command using option --external-cert-file
+
+    Install replica without setup-ca option. On replica install ca
+    using ipa-ca-install with --external-cert-file by providing
+    invalid cert or non-existing file.
+    If executed command with non-existing file, invalid file etc,
+    it should through error.
+
+    related to https://pagure.io/freeipa/issue/6985
+    """
+    num_replicas = 1
+
+    def test_install_ca_replica(self):
+
+        tasks.install_master(self.master)
+        replica = self.replicas[0]
+
+        # install ca with non-existing cert
```

Comment:
Hi @amore17 
in this test the master is installed with a self-signed CA. The CA installation on the replica will have the same self-signed CA and the argument --external-cert-file is ignored.
If the test goal is to check the behavior when --external-cert-file points to a non-existing file, I believe that the test should rather:
- install a CA_less master
- install the replication
- use the 2 step process with ipa-ca-install --external-ca / ipa-ca-install --external-cert-file to install an externally signed CA.